### ISSUE TITLE
protoc: No such file or directory ent\proto\entpb\generate.go:2: running "protoc": exit status 1

### ISSUE THREAD

**Issue by kevinxxq** [ent/ent#1490]
Hi, Sir
I would like to reproduce the project as your blog https://entgo.io/blog/2021/03/18/generating-a-grpc-server-with-ent/
but stuck at the below steps:

_After installing these dependencies, we can re-run code-generation:
go generate ./..._

My command line path:

D:\GoWorks\src\ent-grpc-example>go generate ./...
Could not make proto path relative: entpb\entpb.proto: No such file or directory
ent\proto\entpb\generate.go:2: running "protoc": exit status 1

I checked that the file entpb.proto is there.
Would you please inform me the path to run go generate ./...?

Thank you.

**Comment by rotemtam** [comment]
Hi @kevinxxq , thanks for reaching out!
My first thought is that this is somehow related to Windows path backslashes. I actually haven't tested this flow on Windows, but we can work together to resolve this and fix the tutorial. 

Can you share the contents of ent\proto\entpb\generate.go?
Thanks 🙏

**Comment by kevinxxq** [comment]
Hi Rotem,
Thank you very much for your response.
I just run it again today and it works as expected. It is great guideline for me and I am very excited to see your new version with more functions.

Back to the errors, I guess the cause may be I have installed protoc many locations and it need a include folder for golang(I am working on .Net platform and try to implement it by golang)  and the path has orders to search. I just reorganized the protoc.exe and removed the extra ones.

Thank you for your great work.
